### DISCUSSION TITLE
What do the abbreviated files in the cc module stand for?

### DISCUSSION THREAD

**Discussion by GetPine** [Tsaryii/ProjectOST#2] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/831e5f70-f5b5-48f9-9bef-69439c5db187)
Curious as to what these files stand for, I have looked into them and see what each one does, just curious as to your naming conventions.

**Comment by ColtonKeeley** [comment, marked as answer]
So the Main Directory CC stands for Custom Content.

cc.verse => custom classes
ce.verse => custom enums
cf.verse => custom functions
cfi.verse => custom fx interfaces
ci.verse => custom interfaces
<3
